tap/source/packet_source_manager.go

```diff
@@ -2,200 +2,135 @@ package source
 
 import (
 	"fmt"
-	"runtime"
-	"strconv"
 	"strings"
 
 	"github.com/up9inc/mizu/shared/logger"
-	"github.com/vishvananda/netns"
 	v1 "k8s.io/api/core/v1"
 )
 
+const hostSourcePid = "0"
+
 type PacketSourceManager struct {
-	sources []*tcpPacketSource
+	sources map[string]*tcpPacketSource
 }
 
-func NewPacketSourceManager(procfs string, pids string, filename string, interfaceName string,
+func NewPacketSourceManager(procfs string, filename string, interfaceName string,
 	mtls bool, pods []v1.Pod, behaviour TcpPacketSourceBehaviour) (*PacketSourceManager, error) {
-	sources := make([]*tcpPacketSource, 0)
-	sources, err := createHostSource(sources, filename, interfaceName, behaviour)
-
-	if err != nil {
-		return nil, err
-	}
-
-	sources = createSourcesFromPids(sources, procfs, pids, interfaceName, behaviour)
```

Comment:
Where did this functionality go?
I see that we search for envoy PIDs and Linkerd PIDs, but there is no equivalent for creating sources from some PID (only from host "0" PID).